addons/l10n_in/models/account_invoice.py

```diff
@@ -35,7 +35,12 @@ def _compute_amount_total_words(self):
     @api.depends('partner_id')
     def _compute_l10n_in_gst_treatment(self):
         for record in self:
-            record.l10n_in_gst_treatment = record.partner_id.l10n_in_gst_treatment
+            default_gst_treatment = 'overseas'
+            if record.partner_id.country_id.code == 'IN' and record.partner_id.vat:
+                default_gst_treatment = 'regular'
+            elif record.partner_id.country_id.code == 'IN':
```

Comment:
```suggestion
            elif record.partner_id.country_id.code in ['IN', False]:
```
What happens if no country_id is specified?  It would be weird to suppose overseas then...

Reply:
okay